Dockerfile

```diff
@@ -64,7 +64,24 @@ EXPOSE 80
 
 WORKDIR /etc/httpd
 
-RUN systemctl enable dbus httpd
+## Build Auth-Api
+COPY docker-assets/auth-api /tmp/auth-api
+RUN (cd /tmp/auth-api                                             && \
```

Comment:
I assume most of this will go away once we figure out how to build the RPM for crystal, right? (Part of the reason it's marked as WIP)

Reply:
going back to Ruby, WIP was for that decision as well as testing needed with latest containers. I'll need the initial auth-config container and first IPA config CLI to test end to end, then I can verify this PR and unwip at that time. Thanks.